packages/block-library/src/site-logo/edit.js

```diff
@@ -122,7 +126,7 @@ const SiteLogo = ( {
 
 	const img = (
 		<img
-			className="custom-logo"
+			className={ `custom-logo ${ borderProps.className }` }
```

Comment:
We should use the `classnames` util here and handle when `borderProps.className` is undefined. Otherwise, we'll end up with `custom-logo undefined` as the class name.